packages/core/src/components/button/_button.scss

```diff
@@ -68,6 +68,21 @@ Styleguide components.button.css
     }
   }
 
+  &.pt-loading {
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
Note that using `position: absolute` without setting `top`, `right`, etc. means it gets positioned where it otherwise would have (hence `display: flex`). I can play around with `translate` in a separate commit in a minute